File: Easy/2025-10-13-2273-Find-Resultant-Array-After-Removing-Anagrams/solution.js
var removeAnagrams = function(words) {
    const result = [];
    let prevSorted = '';

    for (const word of words) {
        const sortedWord = word.split('').sort().join('');
        if (sortedWord !== prevSorted) {
            result.push(word);
            prevSorted = sortedWord;
        }
    }

    return result;
};

File: Easy/2025-10-13-2273-Find-Resultant-Array-After-Removing-Anagrams/solution.py
class Solution:
    def removeAnagrams(self, words: List[str]) -> List[str]:
        result = []
        prev_word = ''
        
        for word in words:
            sorted_word = ''.join(sorted(word))
            if sorted_word != ''.join(sorted(prev_word)):
                result.append(word)
            prev_word = word
            
        return result